package.json

```diff
@@ -10,7 +10,7 @@
     "doctoc": "doctoc --maxlevel 3 README.md",
     "jsdoc:widget": "node ./scripts/doc/gen-widget-doc.js",
     "gh-pages": "./scripts/gh-pages.sh",
-    "lint": "eslint . --quiet --no-color && sass-lint --verbose",
+    "lint": "eslint ./components ./decorators ./lib ./widgets --quiet --no-color && sass-lint --verbose",
```

Comment:
Fixed. I incorrectly thought that those files needed to go through babel and that would complexify the build process, but actually they are fine like this.